net/https-dns-proxy/files/https_dns_proxy.init

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh /etc/rc.common
+
+. /lib/functions/network.sh
+
+START=80
+
+USE_PROCD=1
+PROG=/usr/sbin/https_dns_proxy
+
+start_instance() {
+	local cfg="$1"
+	local listen_addr listen_port user group
+
+	config_get listen_addr "$cfg" listen_addr
+	config_get listen_port "$cfg" listen_port
+	config_get user "$cfg" user
+	config_get group "$cfg" group
+
+	procd_open_instance
+	procd_set_param command ${PROG} -d \
```

Comment:
The service should not daemonize itself when `USE_PROCD`.  `procd` needs to keep an eye on its direct child processes to bring them up again (`respawn`).